leshan-server-core/src/main/java/org/eclipse/leshan/server/bootstrap/BootstrapSessionManager.java

```diff
@@ -32,6 +31,29 @@
  */
 public interface BootstrapSessionManager {
 
+    public enum BootstrapAction {
```

Comment:
Not really an "Action". But I cannot propose a better name for now 🙂

Reply:
You have all the week-end to think about that. :wink:

Reply:
Did you find a better name during your week-end :grin:  ?